inc/html.class.php

```diff
@@ -5797,5 +5772,20 @@ static function jsAlertCallback( $msg, $title, $okCallback=null) {
          }).text('".Toolbox::addslashes_deep($msg)."');
          " ;
    }
+
+   /**
+    * Get copyright message as HTML (used in footers)
+    *
+    * @return text
+    */
+   static function getCopyrightMessage() {
+      $message = "<a href=\"http://glpi-project.org/\" title=\"Powered By Teclib\">";
+      $message .= "<span class=\"copyright\">GLPI " .
```

Comment:
I'm not sure about the need for the span which was present only in 2 usage on 4.